gradle/libs.versions.toml

```diff
@@ -103,3 +103,5 @@ spotless = "com.diffplug.spotless:spotless-plugin-gradle:5.12.5"
 subsamplingimageview = "com.davemorrissey.labs:subsampling-scale-image-view:3.10.0"
 
 timber = "com.jakewharton.timber:timber:4.7.1"
+
+play-location = "com.google.android.gms:play-services-location:18.0.0"
```

Comment:
be more explicit name it `google-playservices-location`
also this file is sorted alphabetically and grouped by context